aep/general/0202/aep.md.j2

```diff
@@ -0,0 +1,112 @@
+# Fields
+
+The [`google.api.FieldInfo`][field info proto] type, through its accompanying
```

Comment:
Do you think there's a reasonable generalization of this guidance beyond FieldInfo, such that we could say something about OAS as well (maybe we define an equivalent OAS extension)?

Reply:
I think we should have an equivalent OAS extension. A point of differentiation between AEP + AIP is that we don't have to be as attached to gRPC/Protobuf as AIP is.

Reply:
Works for me, but then IMO we should block this PR on defining that extension in https://github.com/aep-dev/aep.  (We've done similar things elsewhere; e.g. recently we blocked the AEP-ERRORS PR on creation of the ProblemDetails proto.

Reply:
See below, using protovalidate.